pkg/engine/context/context.go

```diff
@@ -5,225 +5,151 @@ import (
 	"strings"
 	"sync"
 
-	"github.com/pkg/errors"
-
 	jsonpatch "github.com/evanphx/json-patch/v5"
-	"github.com/go-logr/logr"
 	kyverno "github.com/kyverno/kyverno/api/kyverno/v1"
+	pkgcommon "github.com/kyverno/kyverno/pkg/common"
+	"github.com/pkg/errors"
 	admissionv1 "k8s.io/api/admission/v1"
 	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
I realized we use mixed loggers across Kyverno packages, would it make sense to choose one and be consistent for logging?

```
"sigs.k8s.io/controller-runtime/pkg/log"
"github.com/go-logr/logr"
```

Reply:
Definitely. IIRC controller runtime uses logr by default.